config/i18n-tasks.yml

```diff
@@ -166,6 +166,10 @@ ignore_missing:
   - users_helper.log_troubleshot
   - users_helper.log_validated
   - users_helper.log_warn
+  - users_helper.log_add_fnok
+  - users_helper.log_remove_fnok
+  - users_helper.log_added_as_fnok
+  - users_helper.log_removed_as_fnok
```

Comment:
Not really sure why we do that for these keys instead of adding them to `config/locales/helpers/en.yml`. Kept current behavior for now but maybe something to change?

Reply:
For a while, we were considering using `default` when doing i18n, but we later changed our minds and just haven't fixed the places where we used that approach. It's fine to leave it for now, though.